test/python/circuit_synthesis/test_gaussian_elimination.py

```diff
@@ -0,0 +1,274 @@
+# Copyright (c) 2023 - 2025 Chair for Design Automation, TUM
+# All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+#
+# Licensed under the MIT License
+
+from __future__ import annotations
+
+import numpy as np
+import pytest
+
+from mqt.qecc.circuit_synthesis.synthesis_utils import CandidateAction, GaussianElimination
+from mqt.qecc.codes.css_code import CSSCode
+
+STEANE_CHECK_MATRIX = np.array([[1, 1, 0, 0, 1, 1, 0], [1, 0, 1, 0, 1, 0, 1], [0, 0, 0, 1, 1, 1, 1]], dtype=np.int8)
+STEANE_CODE = CSSCode(Hx=STEANE_CHECK_MATRIX)
```

Comment:
Since these are only used in one fixture, I would define them in the fixture. 

Also you can get the code with `CSSCode.from_code_name("steane")`